scripts/init_relayers.sh

```diff
@@ -14,14 +21,27 @@ cp ${SCRIPT_DIR}/config/hermes_config.toml $STATE/hermes/config.toml
 cp ${SCRIPT_DIR}/config/relayer_config.yaml $STATE/relayer/config/config.yaml
 
 echo "Adding Hermes keys"
-$HERMES_CMD keys restore --name rly1 --mnemonic "$HERMES_STRIDE_MNEMONIC" $STRIDE_CHAIN_ID 
-$HERMES_CMD keys restore --name rly2 --mnemonic "$HERMES_GAIA_MNEMONIC" $GAIA_CHAIN_ID
+for chain_id in ${CHAINS[@]}; do
+    account_name=$(GET_VAR_VALUE HERMES_${chain_id}_ACCT)
+    mnemonic=$(GET_VAR_VALUE     HERMES_${chain_id}_MNEMONIC)
+
+    $HERMES_CMD keys restore --name $account_name --mnemonic "$mnemonic" $chain_id 
+done
 
 # echo "Adding Relayer keys"
-# $RELAYER_CMD keys restore stride rly1 "$RELAYER_STRIDE_MNEMONIC"
-# $RELAYER_CMD keys restore gaia rly2 "$RELAYER_GAIA_MNEMONIC" 
-# $RELAYER_CMD paths new $STRIDE_CHAIN_ID $GAIA_CHAIN_ID stride-gaia
+# for chain_id in ${chains[@]}; do
```

Comment:
nit: should we rm?

Reply:
I think the go relayers are a quick followup so I'd like to keep it here if you're okay with it